migrate/repos.go

```diff
@@ -93,48 +91,44 @@ func MigrateRepos(source, target *sql.DB) error {
 // value fetched from the source code management system.
 func UpdateRepoIdentifiers(db *sql.DB, client *scm.Client) error {
 	repos := []*RepoV1{}
+	var result error
 
-	// 1. load all repos from the V1 database.
-	err := meddler.QueryAll(db, &repos, "select * from repos where repo_uid LIKE 'temp_%'")
-	if err != nil {
+	if err := meddler.QueryAll(db, &repos, repoTempQuery); err != nil {
 		return err
 	}
 
-	logrus.Infof("updating repository metadata")
+	logrus.Infoln("updating repository metadata")
 
-	var result error
 	for _, repo := range repos {
-		log := logrus.WithField("repository", repo.Slug)
+		log := logrus.WithFields(logrus.Fields{
+			"repo": repo.Slug,
+		})
 
-		// 2.a fetch the repository owner
 		user := &UserV1{}
-		err = meddler.QueryRow(db, user, fmt.Sprintf("SELECT * FROM users WHERE user_id = %d", repo.UserID))
-		if err != nil {
+
+		if err := meddler.QueryRow(db, user, fmt.Sprintf(userIdentifierQuery, repo.UserID)); err != nil {
 			log.WithError(err).Errorf("failed to get repository owner")
 			multierror.Append(result, err)
 			continue
 		}
 
-		log = logrus.WithField("owner", user.Login)
+		log = log.WithField("owner", user.Login)
 
-		// 2.b fetch the remote repository by name.
 		ctx := scm.WithContext(context.Background(), &scm.Token{
 			Token:   user.Token,
 			Refresh: user.Refresh,
 			// Expires: user.Expiry,
 		})
+
 		remoteRepo, _, err := client.Repositories.Find(ctx, scm.Join(repo.Namespace, repo.Name))
+
 		if err != nil {
 			log.WithError(err).Errorf("failed to get remote repository")
 			multierror.Append(result, err)
 			continue
 		}
 
-		// 2.c update the temporary id for the remote
```

Comment:
I think having some of these comments in place are important.  It is probably non-obvious what we are doing here without an explanation